### DISCUSSION TITLE
Is it possible to change pitch axis?

### DISCUSSION THREAD

**Discussion by wiesehahn** [maplibre/maplibre-gl-js#2959] [Q&A]
When changing the pitch usually our reference axis around which the map is rotating is at the map center. Other possibilities (at least in theory) would be that the axis would be in the far distance (e.g. below map) or that axis is at the viewpoint (camera). 

Is there some way to set this behavior?
 (specifically I would like to set the axis a little bit more towards the camera)

**Comment by HarelM** [comment]
You have an offset parameter for the easeTo and flyTo calls, setting this offset will do the trick I believe.
There's also a parameter for the drag handler I think, but I'm not sure.
It should be possible. I think there was either an issue or a discussion that talked about it here in this repo.

**Comment by wiesehahn** [comment]
Thanks, you might mean this one https://github.com/maplibre/maplibre-gl-js/discussions/1989

While we are quite free with `pitch` and `bearing` I dont see any way to go beyond orthogonal view (e.g. pitch -10) to kind of look over the edge and also no way to "lean sideways" to kind of look sideways from the top, right?

**Reply by HarelM** [reply]
I don't think there is as far as I know.
Maybe the support for projections might help, but I'm probably too optimistic.
